Drop-In/src/androidTest/java/com/braintreepayments/api/SelectPaymentMethodFragmentUITest.kt

```diff
@@ -0,0 +1,236 @@
+package com.braintreepayments.api
+
+import androidx.core.os.bundleOf
+import androidx.fragment.app.testing.FragmentScenario
+import androidx.lifecycle.Lifecycle
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.braintreepayments.api.dropin.R
+import org.hamcrest.Matchers.not
+import org.json.JSONObject
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.concurrent.CountDownLatch
+
+// Ref: https://developer.android.com/guide/fragments/test
+@RunWith(AndroidJUnit4::class)
+class SelectPaymentMethodFragmentUITest {
```

Comment:
I'm a big fan of these Kotlin UI tests 👍

Reply:
🙌  Yeah they're way more stable and should allow us to test the UI a lot better.